### PR TITLE
New wizard spell The curse of the barnyard.

### PR DESCRIPTION
**Pull request by Sometinyprick**
Adds a new spell which is essentially the same as the horseman curse except it chooses between three animals. The horse the cow and the pig. The cow mask sprite might be a little iffy also.

Also adds myself to the admins.txt

### CODE REVIEW

**Comment by Chocobro** [comment]
:+1: can't think of much wrong with this.

**Comment by WJohn** [comment]
Why not turn the horseman curse into this?

**Comment by Sometinyprick** [comment]
I wanted to keep it so players could choose between the two.

**Comment by WJohn** [comment]
Still seems pretty redundant, though.

**Comment by Iamgoofball** [comment]
being able to choose is nice

what if I want to be Horsehead Shaman?

**Comment by Incoming5643** [comment]
having just one spell but being able to choose the "flavor" would probably work best, but you know, spellcode

**Comment by Sometinyprick** [comment]
I thought about doing it that way but I thought it would be better if it just picked between the three randomly.

**Comment by Cheridan** [comment]
I really don't think having two spells that have the exact same effect is a good thing...

**Comment by paprka** [comment]
I thought this said 'curse of the baymed'

**Comment by Sometinyprick** [comment]
Huh, should I comment out the horsemask spell? or just replace it with this?
or just keep the original horsemask spell instead of this one

**Comment by Iamgoofball** [comment]
please add some way for us to choose between individual masks

and add a random option too

**Comment by Cheridan** [comment]
@Sometinyprick A poll to determine preferences would be fine.

**Comment by Jordie0608** [comment]
Changelog

File: tgstation.dme [hunk 1]
```diff
@@ -6,6 +6,51 @@
...
+#define FILE_DIR "sound/piano"
+#define FILE_DIR "sound/spookoween"
+#define FILE_DIR "sound/violin"
+#define FILE_DIR "sound/voice"
+#define FILE_DIR "sound/voice/complionator"
+#define FILE_DIR "sound/vox_fem"
+#define FILE_DIR "sound/weapons"
+#define FILE_DIR "tools"
+#define FILE_DIR "tools/AddToChangelog"
+#define FILE_DIR "tools/AddToChangelog/AddToChangelog"
```
Comment (Jordie0608) [inline tgstation.dme]: You'll need to revert this file and stop DM auto-including them:
Build > Preferences > untick 'Automatically set FILE_DIR for sub-directories" > recompile > commit

Comment (Sometinyprick) [inline tgstation.dme]: I added a new file, I thought I was supposed to leave this in If I was doing that?

Comment (RemieRichards) [inline tgstation.dme]: You commit the .dme if you add/remove a file yes, but you still never have FILE_DIR ticked/commited.

File: tgstation.dme [hunk 2]
```diff
@@ -216,6 +261,7 @@
 #include "code\datums\helper_datums\teleport.dm"
 #include "code\datums\helper_datums\topic_input.dm"
 #include "code\datums\spells\area_teleport.dm"
+#include "code\datums\spells\barnyard.dm"
```
Comment (Jordie0608) [inline tgstation.dme]: Down here is where your new file was added correctly, the above FILE_DIR is dream maker being dumb and can be removed.

Comment (Sometinyprick) [inline tgstation.dme]: Alright thanks
I'm going to make a poll for it anyway and if people prefer this I will just edit the horseman curse spell anyway or delete this.
I will keep that in mind if I ever have to add a file in the future.

File: code/datums/spells/barnyard.dm [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		if(!target.unEquip(target.wear_mask))
+			qdel(target.wear_mask)
+		target.equip_to_slot_if_possible(magicpig,slot_wear_mask,1,1)
+
+		flick("e_flash", target.flash)
+		return
+
+	if (prob(55))
+
+		var/obj/item/clothing/mask/horsehead/magichead = new /obj/item/clothing/mask/horsehead
```
Comment (Cheridan) [inline code/datums/spells/barnyard.dm]: This copied code is really bad btw, if you don't know how to fix it then ask in #coderbus

**Comment by Cheridan** [comment]
https://tgstation13.org/phpBB/viewtopic.php?f=10&t=2946

The multi-mask option has pretty clearly won, just waiting on you to update now :)

File: code/datums/spells/barnyard.dm (hunk 3 above)
Comment (Sometinyprick) [inline code/datums/spells/barnyard.dm]: Err using  var/obj/item/clothing/mask/magichead = new (pick(/obj/item/clothing/mask/spig, /obj/item/clothing/mask/cowmask, /obj/item/clothing/mask/horsehead)) is causing no mask to equip.
I have no idea what the issue is but I suppose I will work at it and see if I can figure it out.

**Comment by Sometinyprick** [comment]
Well this should be good to go now

**Pull request merged**